Extensions/PathfindingBehavior/pathfindingruntimebehavior.ts

```diff
@@ -348,7 +346,7 @@ namespace gdjs {
       }
     }
 
-    doStepPreEvents(runtimeScene) {
+    doStepPreEvents(runtimeScene: RuntimeScene) {
```

Comment:
For consistency with the rest of the codebase, I prefer to keep the `gdjs.` prefix everywhere.
It's slightly longer to write/read, but not that much and gives a very clear context (`gdjs.Xxx` => you're in the game engine).
Also make it easy to find types everywhere in the codebase by searching for `: gdjs.RuntimeScene` or even just `gdjs.RuntimeScene`. `RuntimeScene` alone would give too much noise and false positive.
I like this kind of "greppability", important for keeping refactoring and search easy  :)